frontend/src/features/research/context/LeaseContext.tsx

```diff
@@ -0,0 +1,24 @@
+import { ILease } from 'interfaces';
+import { noop } from 'lodash';
+import * as React from 'react';
+import { useState } from 'react';
+
+export interface ILeaseState {
+  lease?: ILease;
+  setLease: (lease: ILease) => void;
+}
+
+export const LeaseStateContext = React.createContext<ILeaseState>({
```

Comment:
Hmm, I'm a bit lost here, why does the research logic need the lease context?

Reply:
Not sure what is going on here. My guess is a bad merge :(